### PR TITLE
docker and hot reloading

### PR DESCRIPTION
**Pull request by dishak**
I have dockerised  the application with bind mount and hotreloading of any ts file changes in backend or web socket folders using nodemon.
@hkirat it would be helpful if you could review this please.

### CODE REVIEW

**Review by siinghd — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+version: '3.8'
+services:
+  postgres:
+    image: postgres
```
[the remark this answers is not in the document]
Comment (dishak) [reply docker-compose.yml]: changed.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    depends_on:
+      postgres:
+        condition: service_healthy
+    ports:
+      - "3000:3000"
+    networks:
+      - my_network
+
+  websocket:
+    image: ws_img:latest
```
[the remark this answers is not in the document]
Comment (dishak) [reply docker-compose.yml]: updated

File: docker-compose.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (dishak) [reply docker-compose.yml]: @siinghd  would mind reviewing please.. thank you..

File: docker-compose.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 version: '3.8'
 services:
   postgres:
-    image: postgres
+    image: postgres:15
```
Comment (siinghd) [inline docker-compose.yml:4]: can we use version 16 here?

**Review by siinghd — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (dishak) [reply docker-compose.yml:4]: > can we use version 16 here?

yes we can but, version 15 is  stable due to it's maturity in market whereas version 16 is just released in sept 2023. But, if I may ask why do you insist on using version 16?

Comment (siinghd) [reply docker-compose.yml:4]: @dishak v16 should be stable also v16 is much faster from what they say. Let's stick to it.

Comment (dishak) [reply docker-compose.yml]: @siinghd sure.. updated it.

Comment (dishak) [reply docker-compose.yml]: @siinghd , Are the PR changes satisfactory? should I close the PR if all good?

**Pull request closed without merging**